### PR TITLE
blog: Add post for Solus 4.6

### PR DESCRIPTION
**Pull request by EbonJaeger**
Depends on #48

### CODE REVIEW

File: content/blog/2024/10/solus-4-6-released/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+### Added Features
+
+- Global Search
+- Enhanced files app
+- Upgraded online accounts, now with OneDrive support
+- Remote login with RDP
+
+### Bug Fixes, cleanup & quality-of-life improvements
+
+- GNOME's file app
```
Comment (Staudey) [inline content/blog/2024/10/solus-4-6-released/index.md:98]: This point seems a bit vague. What does it mean?

File: content/blog/2024/10/solus-4-6-released/index.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+- Over the summer, the Solus team has been tackling the [Usr-Merge conundrum](https://getsol.us/2024/09/04/usr-merge/); read Evan's excellent blog post for the full story. We have now reached the end of Stage 3 on our rollout plan: All users with an updated system now have a Usr-Merge'd system. Any users that create a fresh installation with today's ISOs will also have a merged system. We expect stage 4 to complete Soon™️.
+
+As a reminder, the Usr-Merge brings Solus in line with the compatibility efforts being spearheaded by Fedora and other distributions. More urgently, *systemd* requires the filesystem be merged for version 255.
+
+### Experimental Software Centers
+
+We continue to ship *solus-sc* (Solus Software Center) as the default graphical package manager, but close watchers of Solus development know that we have had experimental support for GNOME Software and KDE Discover included in our repository for many months now.
+
+These software centers let users update software distributed by the Solus repositories, as before, *and* install and manage Flatpaks for third party packages. Flathub is configured as a Flatpak remote by default.
```
Comment (Staudey) [inline content/blog/2024/10/solus-4-6-released/index.md:41]:
```suggestion
These software centers let users update software distributed by the Solus repositories, as before, *and* install and manage Flatpaks for third-party packages. Flathub is configured as a Flatpak remote by default.
```

Merriam-Webster (the preferred dictionary according to the Help Center style guide) hyphenates the adjective form (and it's also written that way further down the text)

File: content/blog/2024/10/solus-4-6-released/index.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+While not installed by default, a couple of other XFCE-related packages have been added since our last release:
+
+- xfdashboard
+- xfce4-battery-plugin
+- xfce4-clipman-plugin
+
+## Known Issues
+
+- [Plasma ISO Requires creation of a kwallet key file before connecting to a network](https://github.com/getsolus/packages/issues/1429)
```
Comment (Staudey) [inline content/blog/2024/10/solus-4-6-released/index.md:147]:
```suggestion
- [Plasma ISO requires creation of a kwallet key file before connecting to a network](https://github.com/getsolus/packages/issues/1429)
```

File: content/blog/2024/10/solus-4-6-released/index.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+- When running `eopkg check`, linux-current and linux-lts may show as broken. These are false positives, and can be disregarded.
+- Users with the proprietary nVidia drivers and the 6.10 kernel may experience a black screen or errors on a black screen after waking the system from sleep.
```
Comment (Staudey) [inline content/blog/2024/10/solus-4-6-released/index.md:149]:
```suggestion
- Users with the proprietary NVIDIA drivers and the 6.10 kernel may experience a black screen or errors on a black screen after waking the system from sleep.
```
It's written in uppercase these days
https://www.nvidia.com/content/dam/en-zz/Solutions/about-us/documents/NVIDIA-Trademark-and-Logo-Usage-Guidelines.pdf

**Review by Staudey — COMMENTED** [review]
Just some very minor things.

(I assume the author field is correct and you're actually just the submitter, as the post also talks about you in the third person)

**Comment by EbonJaeger** [comment]
Correct; David did most of the work. Tracey and I contributed and edited. :D

**Comment by EbonJaeger** [comment]
Sorry @Staudey, this release is not waiting on the blog post :P

I made your changes, anything else can be addressed later.

**Comment by Staudey** [comment]
Huh? I didn't have anything else

**Pull request merged**